concent_api/concent_api/tests/test_middleware.py

```diff
@@ -0,0 +1,51 @@
+from base64                 import b64encode
+import os
+
+from django.test            import override_settings
+from django.test            import TestCase
+from django.urls            import reverse
+from golem_messages         import dump
+from golem_messages         import message
+
+from git import Repo
+
+from utils.testing_helpers  import generate_ecc_key_pair
+
+
+(CONCENT_PRIVATE_KEY,   CONCENT_PUBLIC_KEY)   = generate_ecc_key_pair()
+(PROVIDER_PRIVATE_KEY,  PROVIDER_PUBLIC_KEY)  = generate_ecc_key_pair()
+
+
+@override_settings(
+    CONCENT_PRIVATE_KEY    = CONCENT_PRIVATE_KEY,
+    CONCENT_PUBLIC_KEY     = CONCENT_PUBLIC_KEY,
+)
+class GolemMessagesVersionMiddlewareTest(TestCase):
+
+    def test_golem_messages_version_middleware_attach_http_header_to_response(self):
```

Comment:
Just FYI: our convention with naming tests methods is `test_X_should_Y` where X is what you want to test and Y is what you expect it to do. You're missing the `_should_` here.

Reply:
:+1: